pkg/dyninst/irgen/testdata/snapshot/sample.arch=amd64,toolchain=go1.23.11.yaml

```diff
@@ -11285,6 +11047,16 @@ Types:
                   Variable: {subprogram: 98, index: 3, name: ~r1}
                   Offset: 0
                   ByteSize: 16
+        - Name: val
+          Offset: 33
+          Kind: local
+          Expression:
+            Type: 9 BaseType int
+            Operations:
+                - __kind: LocationOp
+                  Variable: {subprogram: 98, index: 4, name: val}
+                  Offset: 0
+                  ByteSize: 8
```

Comment:
It seems to me like this PR is not trying to introduce a change like this. Some condition you've introduced is changing behaviors. My guess is that we were not computing locals in some situations before and we now are. As we discussed earlier today, I think this sort of thing is a change we want to make, but I don't think it belongs in this PR.

Reply:
Yea the difference was including locals at return. I'll remove for now.